src/map/packets/message_standard.h

```diff
@@ -31,49 +31,69 @@
 /*
  * Valid MessageIDs
  * Complete list at 1/27/76.DAT
+ *
+ * NOTE: System messages do not specify the color in the packet explictly. Instead there is a different message ID
+ * for each of the supported chat colors (system (yellow), say (white), shout (red), party (blue), linkshell (green)).
```

Comment:
Technically the color can be whatever I specify in my client settings. Its more accurate to say the types each have different default colors and that said coloring is a client side thing.

Reply:
Reworded